### ISSUE TITLE
RTCC Maps (DN, RK and MK)

### ISSUE THREAD

**Issue by JoshuaMicallefYBSU** [vatSys/australia-dataset#122]
![SmartSelect_20240723_112508_Discord.jpg](https://github.com/user-attachments/assets/9627d39e-4d23-438d-ae6c-31a816e0bb11)

Discussion from discord. Would be good if these could be included. He is happy to assist as well.

**Comment by zkgell** [comment]
We need a reliable source for these maps

**Comment by Slep-wt** [comment]
Do you think we could FOI these?

**Comment by alphadelta332** [comment]
> Do you think we could FOI these?

My understanding is that in the past, we have sourced our own RTCC data (I have no idea how), and our figures don't quite match up with the IRL RTCC

**Comment by JoshuaMicallefYBSU** [comment]
Given there is no concrete way for AIS to source the data at the own, this will be placed on a burner for the time being.

**Comment by purpletender** [comment]
@JoshuaMicallefYBSU I've spent a couple hours this morning creating a Darwin RTCC, if the map was to be considered, I would be able to finalize it then submit it as a pull request where further refinements could be made. 

<img width="936" height="500" alt="Image" src="https://github.com/user-attachments/assets/51e7b6fa-2776-4865-aa7d-8180b487ef5e" />

In plain english, may I open a pull request for this to be considered for a future release?
